cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Is this something that cypress added?

Reply:
Yes! We can remove it if you like.

Reply:
Yeah, I think it is nice to remove all the boiler-plate things. Thank you!

Reply:
Removed boilerplate files!

Reply:
I also saw a download.html under /cypress/downloads/. I was tempted to remove it but I opened it in chrome and saw some odd symbols so I'm not sure if it's safe to remove; it might be doing something.